### PR TITLE
[Tizen.Network.Bluetooth][Non-ACR] Fix app crash issue during free

### PR DESCRIPTION
**Pull request by Wootak**
Fix app crash issue during free

### CODE REVIEW

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs [hunk 1]
```diff
@@ -177,10 +177,6 @@ internal BluetoothLeDevice(BluetoothLeScanData scanData)
         /// </summary>
         ~BluetoothLeDevice()
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs:178]: Remove empty destructor

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapterImpl.cs [hunk 2]
```diff
@@ -401,32 +401,12 @@ internal IList<BluetoothLeServiceData> GetScanResultServiceDataList(BluetoothLeS
                 _list.Add(BluetoothUtils.ConvertStructToLeServiceData(svc));
             }
 
-            Interop.Libc.Free(serviceListArray);
+            Interop.Bluetooth.FreeServiceDataList(serviceListArray, _serviceListCount);
             Marshal.FreeHGlobal(scanDataStruct.AdvData);
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapterImpl.cs:405]: It seems double free 
It freed on 390 line

Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapterImpl.cs:405]: 390 line is fail case. so, throw exception after free.

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs (hunk 1 above)
Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs:178]: Done.

Comment (Wootak) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs]: How about remaining empty function? in order to remove this, ACR required. but, this function can be used in later.

**Pull request merged**